### PR TITLE
fix: Add githubarj

### PR DESCRIPTION
**Pull request by githubarj**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/1591"></a>

### CODE REVIEW

File: public/data/githubarj.json [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    {
+      "title": "Started open Source",
+      "date": "August 2020",
+      "icon": "github",
+      "color": "grey",
+      "description": "Started Contributing to Open Source",
+      "url": "https://github.com/githubarj"
+    }
+  ]
+}
```
Comment (krshkodes) [inline public/data/githubarj.json:41]: Just a endline is missing :D 
```suggestion
}

```

**Review by krshkodes — CHANGES_REQUESTED** [review]

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge!

**Review by Panquesito7 — APPROVED** [review]
Thanks for contributing! 🎉


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good

**Comment by githubarj** [comment]
Thank you for the correction and approvals.  ^^

**Pull request merged**